docs/src/intro/usage.md

```diff
@@ -76,10 +76,10 @@ As described [here](https://0xpolygonmiden.github.io/miden-vm/intro/overview.htm
 * Secret (or nondeterministic) inputs:
   * `advice_stack` - can be supplied to the VM. There is no limit on how much data the advice provider can hold. This is provided as a string array where each string entry represents a field element.
   * `advice_map` - is supplied as a map of 64-character hex keys, each mapped to an array of numbers.  The hex keys are interpreted as 4 field elements and the arrays of numbers are interpreted as arrays of field elements.
-  * `merkle_store` - the Merkle store is container that allows the user to define `merkle_tree` and `sparse_merkle_tree` data structures.
+  * `merkle_store` - the Merkle store is container that allows the user to define `merkle_tree`, `sparse_merkle_tree` and `partial_merkle_tree` data structures.
     * `merkle_tree` - is supplied as an array of 64-character hex values where each value represents a leaf (4 elements) in the tree.
-    * `sparse_merkle_tree` - is supplied an an array of tuples of the form (number, 64-character hex string).  The number represents the leaf index and the hex string
-    represents the leaf value (4 elements).
+    * `sparse_merkle_tree` - is supplied as an array of tuples of the form (number, 64-character hex string).  The number represents the leaf index and the hex string represents the leaf value (4 elements).
+    * `partial_merkle_tree` - is supplied as an array of tuples of the form ((number, number), 64-character hex string). The internal tuple represents the node depth and index at this depth, and the hex string represents the node value (4 elements).
```

Comment:
Should we say: "The internal tuple represents the leaf depth and index at this depth, and the hex string represents the leaf value (4 elements)."?